sdks/python/apache_beam/io/iobase.py

```diff
@@ -1404,18 +1404,21 @@ class _SDFBoundedSourceWrapper(ptransform.PTransform):
   """
   class _SDFBoundedSourceRestrictionTracker(RestrictionTracker):
     """An `iobase.RestrictionTracker` implementations for wrapping BoundedSource
-    with SDF.
+    with SDF. The tracked restriction is a (SourceBundle, RangeTracker) pair.
+    In order to save bytes sent across the wire, the RangeTracker is set as
+    system tracking RangeTracker only when current_restriction is called.
```

Comment:
This is not only to save bytes across the wire--range trackers are not meant to be serialized. I might say that logically the restriction is a (SourceBundle, RangeTracker) pair, but the second argument is never serialized.

Reply:
Update comment?